compile

```diff
@@ -0,0 +1 @@
+/usr/share/automake-1.16/compile
```

Comment:
Remove this, and the other added files in the souffle root.

Reply:
Done.